_episodes/05-join.md

```diff
@@ -893,6 +903,7 @@ interested in the final match, using both criteria.
 
 ## Transforming coordinates
 
+As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD-1 coordinates. 
```

Comment:
```suggestion
As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD1 coordinates. 
```

I notice we are mixing GD-1 and GD1, is there a reason or should be make it consistent?